.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
We can always improve this file.

Regarding the specific change for md files, I don't think trailing spaces should be an issue.